recipes/ixwebsocket/all/conanfile.py

```diff
@@ -0,0 +1,109 @@
+from conans import ConanFile, CMake, tools
+from conans.tools import Version
+from conans.errors import ConanInvalidConfiguration
+import os
+
+
+class IXWebSocketConan(ConanFile):
+    name = "ixwebsocket"
+    description = "IXWebSocket is a C++ library for WebSocket client and server development"
+    topics = ("conan", "IXWebSocket", "socket", "websocket")
+
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/machinezone/IXWebSocket"
+
+    # License for the library
+    license = "BSD-3-Clause"
+
+    exports_sources = ["CMakeLists.txt"]
+
+    settings = "os", "compiler", "build_type", "arch"
+
+    short_paths = True
+    generators = "cmake"
+
+    options = {
+        "use_mbed_tls": [False, True],
+        "use_tls": [True, False],
+        "use_openssl": [False, True],
+        "use_vendored_third_party": [True, False],
```

Comment:
Remove 'use_vendored_third_party' option. It requires both tls and mbed_tls enabled, but it also requires Perl, Python and zlib to build mbedtls. That option only builds the embedded mbedtls in ixwebsocket.